src/main/kotlin/io.epirus.console/openapi/OpenApiCommand.kt

```diff
@@ -12,24 +12,19 @@
  */
 package io.epirus.console.openapi
 
+import io.epirus.console.EpirusVersionProvider
 import picocli.CommandLine
 import picocli.CommandLine.Command
-import picocli.CommandLine.IVersionProvider
 import picocli.CommandLine.Model.CommandSpec
 import picocli.CommandLine.ParameterException
 import picocli.CommandLine.Spec
-import java.time.Instant
-import java.time.OffsetDateTime
-import java.time.ZoneOffset
-import java.time.format.DateTimeFormatter
-import java.util.Properties
 import java.util.concurrent.Callable
 
 @Command(
     name = "openapi",
     description = ["Generate an OpenAPI project"],
-    versionProvider = OpenApiCommand.VersionProvider::class,
-    subcommands = [GenerateCommand::class],
+    versionProvider = EpirusVersionProvider::class,
```

Comment:
This command shouldn't need a version, only Epirus will.

This should be removed.

Reply:
done